spec/migrations/20170410130134_copy_server_id_from_jobs_to_miq_tasks_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_migration
+
+describe CopyServerIdFromJobsToMiqTasks do
+  let(:task_name) { "Hello Test Task" }
+  let(:task_stub) { migration_stub(:MiqTask) }
+  let(:server_id) { 1111 }
+
+  migration_context :up do
+    it "copies data from 'jobs.miq_server_id' to 'miq_tasks.miq_server_id'" do
+      task = task_stub.create!(:name => task_name)
+      migration_stub(:Job).create!(:miq_server_id => server_id, :miq_task_id => task.id)
+      migrate
```

Comment:
Same comments from the other PR 

- Blank lines around migrate
- create a let for the migration_stub(:Job)